src/scm/bitbucket.js

```diff
@@ -196,111 +203,126 @@ class Bitbucket {
       .then(() => {
-        return repo;
+        return $.ajax({
+          url: `${this.baseUrl}/repositories/${owner}/${name}`,
+          headers: {
+            'Authorization': `Bearer ${this.accessToken}`
+          },
+          method: 'POST',
+          crossDomain: true,
+          dataType: 'json',
+          contentType: 'application/json',
+          data: JSON.stringify(payload)
+        })
+      })
+      .then(response => {
+        const repo = {
+          fullName: response.full_name
+        };
+        context.repo = repo;
+        Object.assign(context.bindRepo, {
+          [context.id]: repo
+        });
+        if (context.bindBranch[context.id]) {
+          delete context.bindBranch[context.id];
+        }
+        chrome.storage.sync.set({
+          bindRepo: context.bindRepo
+        });
+        return response.full_name;
+      })
+      .then(repo => {
+        return this.commitFiles(repo, 'master', null, [{
+            name: "README.md",
+            content: "initialed by gas-github"
+          }], null, 'initial commit')
+          .then(() => {
+            return repo;
+          });
+      })
+      .catch((err) => {
+        throw new Error('Failed to create new repository.');
       });
-    })
-    .catch((err) => {
-      throw new Error('Failed to create new repository.');
-    });
   }
 
   createBranch() {
     const branch = $('#new-branch-name').val();
     if (!branch || branch === '') return;
     return this.getAccessToken()
-    .then(() => {
-      return $.getJSON(
-        `${this.baseUrl}/repositories/${context.repo.fullName}/refs/branches/${context.branch}`,
-        { access_token: this.accessToken }
-      );
-    })
-    .then(res => {
-      const parent = res.target? res.target.hash : null;
-      return this.commitFiles(context.repo.fullName, branch, parent, [], null, `create new branch ${branch}`);
-    })
-    .then(() => {
-      context.branch = branch;
-      Object.assign(context.bindBranch, { [context.id] : branch });
-      chrome.storage.sync.set({ bindBranch: context.bindBranch });
-      return branch;
-    })
-    .catch(err => {
-      throw new Error('Failed to create new branch.');
-    });
+      .then(() => {
+        return $.getJSON(
+          `${this.baseUrl}/repositories/${context.repo.fullName}/refs/branches/${context.branch}`, {
+            access_token: this.accessToken
+          }
+        );
+      })
+      .then(res => {
+        const parent = res.target ? res.target.hash : null;
+        return this.commitFiles(context.repo.fullName, branch, parent, [], null, `create new branch ${branch}`);
+      })
+      .then(() => {
+        context.branch = branch;
+        Object.assign(context.bindBranch, {
+          [context.id]: branch
+        });
+        chrome.storage.sync.set({
+          bindBranch: context.bindBranch
+        });
+        return branch;
+      })
+      .catch(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/scm/bitbucket.js#L279"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>